akka-sse/src/main/scala/de/heikoseeberger/akkasse/LineParser.scala

```diff
@@ -34,15 +34,15 @@ private final class LineParser(maxLineSize: Int) extends GraphStage[FlowShape[By
 
   private val out = Outlet[String]("line-parser.out")
 
-  override val shape = FlowShape.of(in, out)
-
-  private var buffer = ByteString.empty
+  override val shape = FlowShape(in, out)
 
   override def createLogic(inheritedAttributes: Attributes) = new GraphStageLogic(shape) {
+    private var buffer = ByteString.empty
     setHandler(in, new InHandler {
       override def onPush() = {
-        def parseLines(): Vector[String] = {
+        def parseLines(buffer: ByteString): (ByteString, Vector[String]) = {
```

Comment:
You're not the sole reader of it, I know because I just read it and had to go into the impl to find out :)